platform/engineering/codeowners.md

```diff
@@ -1,23 +1,28 @@
 
-After creating your GitHub team, follow the conventions in the [`CODEOWNERS` configuration file](https://github.com/department-of-veterans-affairs/vets-website/blob/master/.github/CODEOWNERS) to assign your team to an application folder. You should understand the following when making changes to the code owners configuration: 
+In `vets-website`, a VSA team can be assigned as a code owner of [application folders](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications). **This will remove `frontend-review-group` as a required reviewer when changes are made to those folders.** Follow the conventions in the [`CODEOWNERS` configuration file](https://github.com/department-of-veterans-affairs/vets-website/blob/master/.github/CODEOWNERS) to assign your team to an application folder.
+
+In `vets-api`, a VSA team can be assigned as a code owner of [module folders](https://github.com/department-of-veterans-affairs/vets-api/tree/master/modules). **This will remove `backend-review-group` as a required reviewer when changes are made to those folders.** Follow the conventions in the [`CODEOWNERS` configuration file](https://github.com/department-of-veterans-affairs/vets-api/blob/master/.github/CODEOWNERS) to assign your team to an application folder.
+
+You should understand the following when making changes to the code owners configuration:
 
 - The **last matching rule** that matches all files in a change **takes precedence**.
-- Application code should be structured to **require as few rules as possible**. 
-- Only one required reviewer per file can be triggered by code owners rules. A pull request may require multiple code owner approvals. 
-- Code owners rules are applied to every push after a PR is created. Required reviewers can be removed during this process but **GitHub does not remove the frontend review group once it has been added to a PR** as a required reviewer. 
-- The `frontend-review-group` approves changes to the `CODEOWNERS` configuration. The `frontend-review-group` will add new GitHub teams created by VSA teams to the `vets-website` repository during this review since this is required for code owners rules to trigger. 
+- Application code should be structured to **require as few rules as possible**.
+- Only one required reviewer per file can be triggered by code owners rules. A pull request may require multiple code owner approvals.
+- Code owners rules are applied to every push after a PR is created. Required reviewers can be removed during this process but **GitHub does not remove a review group once it has been added to a PR** as a required reviewer.
+- The VSP teams approve changes to the `CODEOWNERS` configuration. The relevant VSP team will add new GitHub teams created by VSA teams to the applicable repository during this review since this is required for code owners rules to trigger.
```

Comment:
Do we want to include something about rare cases where code owners can be overridden and how to go about that process?

Reply:
A few lines above says "- The **last matching rule** that matches all files in a change **takes precedence**."  Is that what you mean or is there another process you're referring to?

Reply:
I'm thinking about the scenario where a codeowner is blocking a merge and the edge case where it needs to be overridden. Not sure if this applies here, but it came to mind when I was remembering how some of the Zeitwerk changes were blocked by various codeowners

Reply:
Ah!  That scenario doesn't exist anymore because backend-review-group is a global code owner. https://github.com/department-of-veterans-affairs/vets-api/blob/7360d520f2dfdd7b5a902139eaea44cef967b103/.github/CODEOWNERS#L6

So "sub-teams" can't block us from merging in the future